tests/Extension/OptionalChainingTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+// tests/Extension/OptionalChainingTest.php
+
+namespace Twig\Tests\Extension;
+
+use Twig\Test\IntegrationTestCase;
+
+class OptionalChainingTest extends IntegrationTestCase
```

Comment:
There is no `OptionalChaining` extension. The operator is registered by CoreExtension. This should be tested by the existing integration tests by putting the fixtures in the existing folder.